utils/validate-data-source.js

```diff
@@ -1,13 +1,30 @@
 const appRoot = require('app-root-path');
 
-const { validateFilePath } = appRoot.require('api/v1/db/json/fs-operations');
-
-const dbPath = 'tests/unit/mock-data.json';
+const { validateDBPath } = appRoot.require('api/v1/db/json/fs-operations');
+const { validateDBConnection } = appRoot.require('api/v1/db/oracledb/connection');
 
 /**
  * @summary Validate database configuration
+ * @param {string} dataSourceType data source type
  * @function
```

Comment:
Very minor detail, but I think we usually put `@function` directly under `@summary` based on our other examples.